101-backup-vault-basic/azuredeploy.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vaultName": {
+      "type": "string",
+      "metadata": {
```

Comment:
does this need to be globally unique?  If so, add that to the description and provide a defaultValue using uniqueString()

Reply:
should be unique within resource group and location. will provide appropriate default value in that case.

Reply:
added default value.